velox/exec/tests/TableWriteTest.cpp

```diff
@@ -137,7 +138,7 @@ struct TestParam {
 
   CompressionKind compressionKind() const {
     return static_cast<facebook::velox::common::CompressionKind>(
-        (value & ((1L << 48) - 1)) >> 40);
+        (value & ((1L << 56) - 1)) >> 48);
```

Comment:
To be removed later because this is addressed in a different PR which is out for review.

Reply:
That PR got merged. Let's remove it.